Bottom: fb48a6ffdf5ff0e91f3d38f3bdee789c57f42a05
Top:    30db318b81f2de103d54a40506ec69e9105592e1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 17:56:57 +0100

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index 75e1b9e..c0dee55 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -71,6 +71,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 566596d..3d2e763 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -100,7 +101,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
